crates/bevy_pbr/src/render/mesh_view_bindings.rs

```diff
@@ -213,11 +213,13 @@ fn layout_entries(
                 ))]
                 texture_cube(TextureSampleType::Depth),
             ),
-            // Point Shadow Texture Array Sampler
+            // Point Shadow Texture Array Comparison Sampler
             (3, sampler(SamplerBindingType::Comparison)),
+            // Point Shadow Texture Array Linear Sampler
+            (4, sampler(SamplerBindingType::Filtering)),
             // Directional Shadow Texture Array
             (
-                4,
+                5,
```

Comment:
Not a change for this PR, but after this is merged would a PR be welcome to replace these values with constants? and defined in the shader with defines like this:
https://github.com/bevyengine/bevy/blob/4a312a12e79ccf32b54ba2d3e99053685028c055/crates/bevy_core_pipeline/src/tonemapping/lut_bindings.wgsl#L3-L4

It might reduce merge conflict churn when there's lots of unrelated changes in a PR.

Reply:
As I recall, non-boolean constants are kind of annoying in our PBR shaders at the moment, but sure, it could be done in the future.